### ISSUE TITLE
testing 4543: changing picture size in CKeditor doesn't effect frontend

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3821]
**URL:** https://meinberlin-dev.liqd.net/vorhaben/2021-00074/
**user:** any
**expected behaviour:** if I change the size of my picture in CKeditor the size is also changed in frontend
**behaviour:** picture always has the same size, changes are not saved
**important screensize:**
**device & browser:** chrome, big sur
**Comment/Question:** Is that on purpose because we want the plan to look nice and not have screwed up pictures? I think we should allow it as we also do that for projects.

Screenshot?

**Comment by CarolingerSeilchenspringer** [comment]
@khamui this still does not work. Am I missing something?

**Comment by khamui** [comment]
@CarolingerSeilchenspringer sorry why bad. I linked this issue, but it was not solved by the PR back then.

@fuzzylogic2000 @Rineee @phillimorland @sabinammm : I investigated and figured out that the `<img>'s ` attributes `width` and `height` are saved to backend (POST), but in case of plans the GET does not return the image with those set attributes. Maybe someone can pair with me having a look how to check this?
